pyiron_atomistics/atomistics/structure/factories/ase.py

```diff
@@ -36,19 +36,19 @@ def ase_wrapped_doc(f_name, chain):
     @wraps(ase_bulk)
     def bulk(self, *args, **kwargs):
         s.publication_add(publication_ase())
-        return ase_bulk(*args, **kwargs)
+        return ase_to_pyiron(ase_bulk(*args, **kwargs))
     bulk.__doc__ = ase_wrapped_doc('bulk','ase.build.bulk') + bulk.__doc__
 
     @wraps(ase_cut)
     def cut(self, *args, **kwargs):
         s.publication_add(publication_ase())
-        return ase_cut(*args, **kwargs)
+        return ase_to_pyiron(ase_cut(*args, **kwargs))
```

Comment:
I think as @sudarsan-surendralal and @niklassiemer pointed out in [#76](https://github.com/pyiron/pyiron_atomistics/pull/76#discussion_r579790057), there is no need to use `ase_to_pyiron` for `cut` and `stack`.
```suggestion
        return ase_cut(*args, **kwargs)
```

Reply:
Yes, that was also my understanding.